service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RepresentativesController.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.annotations.Api;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Others;
+import uk.gov.hmcts.reform.fpl.service.OthersService;
+import uk.gov.hmcts.reform.fpl.service.RepresentativeService;
+import uk.gov.hmcts.reform.fpl.service.RespondentService;
+
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+
+@Api
+@RestController
+@RequestMapping("/callback/manage-representatives")
+public class RepresentativesController {
+
+    private final ObjectMapper mapper;
+    private final RepresentativeService representativeService;
+    private final RespondentService respondentService;
+    private final OthersService othersService;
+
+    @Autowired
+    public RepresentativesController(ObjectMapper mapper,
```

Comment:
shall we replace that with `@RequiredArgsConstructor(onConstructor = @__(@Autowired))`?

Reply:
done